cartographer/io/migrate_serialization_format_main.cc

```diff
@@ -46,7 +48,8 @@ int main(int argc, char** argv) {
             << FLAGS_original_pbstream_file
             << "\" to new serialization format in \""
             << FLAGS_output_pbstream_file << "\"";
-  cartographer::io::MigrateStreamFormatToVersion1(&input, &output);
+  cartographer::io::MigrateStreamFormatToVersion1(&input, &output,
+                                                  FLAGS_uses_old_submap);
```

Comment:
Would be good if you could wait for #1326 to go in and then add the flag to the internal/pbstream_migrate.cc
Should be only a minor conflict to resolve hopefully.